tests/ci-csi-manila-e2e.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
```

Comment:
This script is copy-paste from csi-cinder's e2e job. I'd prefer to factor this out into a separate file and keep the scripts for the individual e2e entry points minimal. Maybe in a separate PR?

Reply:
+1